tests/unittests/CoreGraphics/CGPathTests.mm

```diff
@@ -14,7 +14,224 @@
 //
 //******************************************************************************
 
-#import <TestFramework.h>
-#import <Starboard.h>
 #import <CoreGraphics\CGPath.h>
 #import <Foundation\Foundation.h>
+#import <Starboard.h>
+#import <TestFramework.h>
+
+static NSString* const kPointsKey = @"PointsKey";
+static NSString* const kTypeKey = @"TypeKey";
+
+int cgPathPointCountForElementType(CGPathElementType type) {
+    int pointCount = 0;
+
+    switch (type) {
+        case kCGPathElementMoveToPoint:
+        case kCGPathElementAddLineToPoint:
+            pointCount = 1;
+            break;
+        case kCGPathElementAddQuadCurveToPoint:
+            pointCount = 2;
+            break;
+        case kCGPathElementAddCurveToPoint:
+            pointCount = 3;
+            break;
+        case kCGPathElementCloseSubpath:
+            pointCount = 0;
+            break;
+    }
+    return pointCount;
+}
+
+// CGPathApplierFunction that attempts to modify elements inside a CGPath
+void cgPathModifyingApplierFunction(void* info, const CGPathElement* element) {
+    int pointCount = cgPathPointCountForElementType(element->type);
+
+    NSMutableArray* points = [NSMutableArray array];
```

Comment:
Are you referring to the points NSMutableArray not being used? It's not used, I can remove it. cgPathModifyingApplierFunction just exists to show that paths cannot be manipulated in an applier function.